docs/developers/upgrading.md

```diff
@@ -18,6 +18,15 @@ on our Github repo.
 
 Your current version can be found in `Pupilfirst::Application::VERSION` or in the Docker image tag.
 
+### 2023.5
```

Comment:
This version upgrade wouldn't be required as `submission_details` query is not exposed for use through API and is only used internally.

cc: @harigopal